YouTubeChannels.md

```diff
@@ -69,6 +69,10 @@
 
 * [Google Developers](https://www.youtube.com/user/GoogleDevelopers): The Google Developers channel offers some lessons, tutorials, and guides on various development topics. However, the most focus is on product reviews, recorded talks from tech experts at Google events, and current industry news that are really important for developers.
 
+* [Eddie Joude](https://www.youtube.com/user/eddiejoude): Do you want to get into coding, the community and open source? Do you want to make more money, have flexible and remote working? Subscribe now to my channel and watch my videos to learn how.
```

Comment:
@FayasNoushad, could you please correct Eddie's surname to `Jaoude` in both the array brackets and the url please?

Reply:
Ohh! Sorry my mistake. Its fixed now